fix: update custom secret howto (#385)
ory/k8s: docs/helm/kratos.md

> ## Configuration

Reviewer: ```suggestion
## Kratos configuration
```

$ helm install -f ./path/to/kratos-config.yaml ory/kratos
```

## Helm Chart Configuration

> Have a look at the [Ory Kratos Helm Chart README](https://github.com/ory/k8s/tree/master/helm/charts/kratos) for the full list of Helm Chart values.

Reviewer: ```suggestion
For the full list of chart values, please refer to the [Ory Kratos Helm Chart README](https://github.com/ory/k8s/tree/master/helm/charts/kratos), or the [values.yaml](https://github.com/ory/k8s/blob/master/helm/charts/kratos/values.yaml) directly 
```

> Additionally, the following extra settings are available:

Reviewer: ```suggestion
## Helm chart configuration
Additionally, the following extra settings are available:
```

Reviewer: I think we could benefit from adding an extra header in the docs, and point to the full list of helm values and their description (so the readme of the chart)

Author: Done :), let me know what you think.
